From 94769e8414c653a1339b693bec86a6b2d0fef06c Mon Sep 17 00:00:00 2001
From: Shawn Xiao <b49994@freescale.com>
Date: Mon, 7 Dec 2015 10:54:45 +0800
Subject: [PATCH 0759/5242] MGS-1087: ARM: dts: Add CMA setting to 7d dtsi

commit  fc1c370a28493f7a1be2bf84ab77b8f6454f2812 from
https://source.codeaurora.org/external/imx/linux-imx.git

Since the cma setting has been moved to dts file. Add this partation to
7d dtsi to make cma pool building successful on 7d soc.

Signed-off-by: Shawn Xiao <b49994@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d.dtsi |   14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 3e19e51..30bc5ca 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -47,6 +47,20 @@
 		#phy-cells = <0>;
 	};
 
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		/* global autoconfigured region for contiguous allocations */
+		linux,cma {
+			compatible = "shared-dma-pool";
+			reusable;
+			size = <0x14000000>;
+			linux,cma-default;
+		};
+	};
+
 	soc {
 		etm@3007d000 {
 			compatible = "arm,coresight-etm3x", "arm,primecell";
-- 
1.7.9.5

